docs/source/user/turbsim/output.rst

```diff
@@ -0,0 +1,33 @@
+.. _TurbSim_output:
+
+Output Files
+============
+
+
+HAWC Full-Field Files
+---------------------------------------------------
+
+When TurbSim is requested to write HAWC-formatted output files (``WrHAWCFF=TRUE``), it will generate four files. ``<RootName>-u.bin``, 
+``<RootName>-v.bin``, and ``<RootName>-w.bin`` are binary files that contain the full-field turbulence data for the 3 wind-speed components.
+``<RootName>.HAWC`` is a text summary file that indicates the number of points in the binary files and how they should be scaled. The data 
+in this file is written in a format that can be copied into a HAWC2 input file.
+
+
+Notes: 
+
+1. The ``factor_scaling`` values in the summary file indicate the inverse of the values TurbSim used to scale the data in the HAWC files.
+   ``factor_scaling`` can theoretically be used in HAWC2 to obtain the original data generated by TurbSim.
+   TurbSim scales the data so that HAWC2 will obtain the standard-deviation ratios of 1.0 (u/u), 0.8 (v/u), and 0.5 (w/u) as a work-around for
+   an issue in how HAWC2 scales turbulence files. Please note that these ratios may not work well with non-IEC turbulence models.
+
+2. HAWC-formatted files are always periodic, so all of the analysis time steps are written.
+   
+3. The u-component wind speed files have the mean *hub-height* wind speed removed, so they will contain any shear that was defined.
```

Comment:
Presumably you mean "The u-component wind speed files have the mean *hub-height* wind speed removed; they will contain any shear that was defined minus this mean."?

Reply:
See #1178 for correction